addons/Pouches/Pouches.lua

```diff
@@ -51,11 +54,21 @@ end)
 function use_item()
     windower.chat.input('/item "' .. item.name .. '" <me>')
     item.count = item.count - 1
-    if item.count > 0 and windower.ffxi.get_player().status == 0 then
-        use_item:schedule(item.delay)
+    if item.count > 0 and windower.ffxi.get_player().status == 0 then	
+        if inventory_info.current < inventory_info.max then 
+            use_item:schedule(item.delay)
+        else
+            log("Your inventory is full...Aboriting")
+		  end
```

Comment:
Replace the tabs with 4 spaces (elsewhere too, if I missed it, can just do a blanket replace).